packages/i18n/package.json

```diff
@@ -12,8 +12,10 @@
   "type": "module",
   "dependencies": {
     "@wordpress/i18n": "^3.17.0",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
     "prop-types": "^15.7.2"
+  },
+  "devDependencies": {
+    "react": "^16.14.0",
```

Comment:
`react` itself should stay in `dependencies` as it's used in the source files.

`react-dom` is only used in tests, so should be in `devDependencies`

Reply:
Fixed